doc/chips/10.rst

```diff
@@ -250,6 +250,78 @@ block while the rules for Phase 2 may be applied to the second, without a
 constant check to the location of the ``.init()`` call which is the linchpin for
 syntax 1.
 
+Child's fields could depend on parent's fields
+**********************************************
+
+We also considered allowing child's fields to be initialized based on
+parent's fields. For example, the child could control where, in its Phase 1,
```

Comment:
"a child's fields", "its parent's fields".  Missing articles will make the document seem less formal.